internal/mock/mock_deviceclient.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	InvalidDeviceId   = "1ef435eb-5060-49b0-8d55-8d4e43239800"
+	InvalidDeviceId = "1ef435eb-5060-49b0-8d55-8d4e43239800"
```

Comment:
whitespace only changes in this file (see above)!

Reply:
white space changed by go fmt